android/KMEA/app/src/main/java/com/keyman/engine/KMKeyboard.java

```diff
@@ -232,7 +232,13 @@ public void initKMKeyboard(final Context context) {
 
     getSettings().setUseWideViewPort(true);
     getSettings().setLoadWithOverviewMode(true);
-    if (0 != (context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE)) {
+
+    // When `.isTestMode() == true`, the setWebContentsDebuggingEnabled method is not available
+    // and thus will trigger unit-test failures.
+    if (!KMManager.isTestMode() && (
+      0 != (context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE)
+      || KMManager.getTier(null) != KMManager.Tier.STABLE
+    )) {
```

Comment:
```suggestion
    if (!KMManager.isTestMode() && (
      (context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE) != 0 || 
      KMManager.getTier(null) != KMManager.Tier.STABLE
    )) {
```

seriously only a nit, but IMO a little easier to read if we:
1. keep operators in the same place on the line
2. keep constants in the same place on the line (most of our code has placed constants on the RHS of the comparison; I know the argument with = vs == but let's not go there today)